### ISSUE TITLE
CJK fonts don't show after with output()

### ISSUE THREAD

**Issue by mxts** [dompdf/dompdf#1980]
Hi,

I have a PDF with CJK characters using ```@font-face```, everything is fine.
However, I wish to add page numbers and the examples I have seen calls ```$pdf->output``` prior to ```page_text()``` or ```page_script()```. Once ```$pdf->output``` is called, the resulting CJK characters do not show in the PDF.

Any ideas? Thank you.

**Comment by bsweeney** [comment]
So did you figure it out?

**Comment by bilogic** [comment]
no, the issue remains and I did not have more time to dig further, gist of the issue is:

I'm using ```laravel-dompdf``` to create a PDF with CJK characters using @font-face, everything is fine.

However, I wish to add page numbers and the examples I have seen calls ```$pdf->output``` prior to ```page_text()``` or ```page_script()```. Once ```$pdf->output``` is called, the resulting CJK characters do not show in the PDF.

**Comment by bsweeney** [comment]
Do you still have the problem if you disable font subsetting (assuming it's enabled)?

**Comment by bilogic** [comment]
Yes, it happens both enabled/disabled.

**Comment by bsweeney** [comment]
So what order are you calling things? Once Dompdf compiles the PDF things change internally such that further manipulation won't work. Also, are you using encryption?
